.travis.yml

```diff
@@ -4,6 +4,7 @@ addons:
   apt:
     packages:
       - libenchant-dev
+      - python3.6
```

Comment:
I wish we could the travis-ci matrix feature. See https://github.com/Kinto/kinto/blob/master/.travis.yml#L29-L33